git-hooks/commit/index.js

```diff
@@ -0,0 +1,7 @@
+const TYPES = require('./types')
```

Comment:
Please see comment above concerning `semicolons`. 

I recommend adding the semicolons.